app/src/main/res/values/strings.xml

```diff
@@ -80,6 +80,11 @@
     <string name="setting_message_layout_dialog_button2">Later</string>
     <string name="setting_time_format">Time format</string>
     <string name="setting_key_time_format">time_format</string>
+    <string name="setting_notifications">Notifications</string>
+    <string name="setting_notification_channels">Separate app notification channels</string>
+    <string name="setting_key_notification_channels">notification_channels</string>
+    <string name="setting_notification_channels_dialog_title">Manual refresh required</string>
+    <string name="setting_notification_channels_dialog_message">For the change to take effect you must manually refresh the apps using the refresh button in the main menu.</string>
```

Comment:
We could do the same as it's done for the theme switch, force a restart of the whole android app. See `com.github.gotify.settings.SettingsActivity.SettingsFragment#onViewCreated`

Reply:
But a restart does not update the existing apps, right?

Reply:
Depends on what you mean with "update the existing apps", gotify generally doesn't cache anything besides maybe images between app restarts, so everything will be refetched.

Reply:
Oh okay, didn't know that.
Then we can adapt the restart dialog, of course.

Reply:
Resolved in 9d2622fbc6fe1d827422bbd3b18bce4046be5294